src/main/java/uk/gov/hmcts/reform/unspec/launchdarkly/OnBoardingOrganisationControlService.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.unspec.launchdarkly;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.launchdarkly.sdk.LDValue;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.prd.model.Organisation;
+import uk.gov.hmcts.reform.unspec.service.OrganisationService;
+import uk.gov.hmcts.reform.unspec.service.UserService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class OnBoardingOrganisationControlService {
+
+    public static final String ORG_NOT_REGISTERED = "your organisation is %s, unfortunately that org "
+        + "is not part of pilot and therefore cannot use Civil Damages service";
+    
+    private final FeatureToggleService featureToggleService;
+    private final OrganisationService organisationService;
+    private final ObjectMapper objectMapper;
+    private final UserService userService;
+
+    public List<String> isOrganisationAllowed(String userBearer) {
```

Comment:
would rename this method as does not return boolean

Reply:
renamed